services/server/src/server/services/storageServices/AbstractDatabaseService.ts

```diff
@@ -442,7 +459,7 @@ export default abstract class AbstractDatabaseService {
       }
 
       if (
-        (hasCreationAuxdataTransformation &&
+        (!existingVerifiedContract.creation_metadata_match &&
```

Comment:
This is not related to this PR, but I noticed that we were still using transformations to check if an existing match needs to be upgraded